unit_tests/test_issue_879.py

```diff
@@ -0,0 +1,23 @@
+import os
+import unittest
+from nose.plugins.skip import SkipTest
+from nose import commands
+
+
+class TestIssue879(unittest.TestCase):
+    def setUp(self):
+        try:
+            import setuptools
+        except ImportError:
+            raise SkipTest("setuptools not available")
+
+    def test_can_ignore_user_config_files(self):
+        support = os.path.join(
+            os.path.dirname(__file__), 'support', 'issue879')
+        os.environ['HOME'] = support
+        os.environ['NOSE_IGNORE_CONFIG_FILES'] = '1'
```

Comment:
I think this is probably better done as a functional_test.  This is modifying the processes environment which is used to run all the tests.  While everything passes, I'd feel more comfortable launching it as a separate process, which the functional tests would do.